### PR TITLE
Posman

### PR DESCRIPTION
**Pull request by Anntroy**

### CODE REVIEW

File: data_converter/settings_local.base.py [hunk 1]
```diff
@@ -10,6 +10,12 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
+# AUTHORIZATION TOKEN FOR POSTMAN
+POSTMAN_TOKEN = 'e7937e4b5a309177b85fad8715bcbd451fdcb67f91696e1968f8dd188ada70a1'
+
+# URL FOR POSTMAN
+POSTMAN_URL = 'https://postman.org.ua/api/v1/email/'
```
Comment (roman-tiukh) [inline data_converter/settings_local.base.py:17]:
```suggestion
POSTMAN_URL = 'https://postman.org.ua/'
```

File: data_ocean/postman.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+def send_registration_email(email, first_name):
+    header_obj = {
+        "Content-Type": "application/json",
+        "Authorization": f"Token {settings.POSTMAN_TOKEN}",
+    }
+    data_obj = {
+        "recipient": email,
+        # "template": "59_email_data_ocean_registration"
+        "text": f"{first_name}, ви успішно зареєстровані у системі Data Ocean.\n\r Якщо Вами такі дії не проводились, проігноруйте цей лист.\n\r З повагою, Data Ocean",
```
Comment (roman-tiukh) [inline data_ocean/postman.py:13]:
```suggestion
        "text": f"{first_name}, ви успішно зареєстровані у системі Data Ocean.\r\n Якщо Вами такі дії не проводились, проігноруйте цей лист.\r\n З повагою, Data Ocean",
```

File: users/models.py [hunk 3]
```diff
@@ -50,3 +53,15 @@ class DataOceanUser(AbstractUser):
...
+        header_obj = {
+            'Content-Type': 'application/json',
+            'Authorisation': settings.POSTMAN_TOKEN,
+            }
+        data_obj = {
+            "recipient": self.email,
+            "text": self.first_name + ", ми отримали запит на реєстрацію у системі Data Ocean.",
+            "subject": "Підтвердження реєстрації на Data Ocean"
+            }
+        return requests.post(settings.POSTMAN_URL, data=data_obj, headers=header_obj)
```
Comment (roman-tiukh) [inline users/models.py]: what is this?

File: users/models.py [hunk 4]
```diff
@@ -1,8 +1,11 @@
 from django.db import models
 from django.contrib.auth.models import AbstractUser
 from django.contrib.auth.base_user import BaseUserManager
+from django.conf import settings
 from django.utils.translation import ugettext_lazy as _
 
+import requests
+
```
Comment (roman-tiukh) [inline users/models.py]: unused import

File: data_ocean/postman.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+        "subject": "Підтвердження реєстрації на Data Ocean"
+    }
+    response = requests.post(settings.POSTMAN_URL, json=data_obj, headers=header_obj)
```
Comment (roman-tiukh) [inline data_ocean/postman.py:16]:
```suggestion
    response = requests.post(f"{settings.POSTMAN_URL}api/v1/email/", json=data_obj, headers=header_obj)
```

File: users/models.py (hunk 3 above)
Comment (Anntroy) [reply users/models.py]: I've deleted it all from users model

File: users/models.py (hunk 4 above)
Comment (Anntroy) [reply users/models.py]: deleted

**Pull request merged**